readme.md

```diff
@@ -116,6 +116,7 @@ It only needs the first 4100 bytes.
 - [`amr`](https://en.wikipedia.org/wiki/Adaptive_Multi-Rate_audio_codec)
 - [`pdf`](https://en.wikipedia.org/wiki/Portable_Document_Format)
 - [`epub`](https://en.wikipedia.org/wiki/EPUB)
+- [`mobi`](https://wiki.mobileread.com/wiki/MOBI#Format) - Mobipocket
```

Comment:
I should have been clearer. I just meant link it as a comment here. This should still point to the Wikipedia article.

Reply:
Ah, I see! I should have seen all the other Wikipedia links.